packages/website/docs/Checkbox.md

```diff
@@ -1,29 +1,57 @@
--   If more than 7 options (checkboxes) are needed, it is preferable to group checkboxes logically under more than one set of options such that the set label represents the grouping logic.
+Use a stand-alone checkbox to toggle a single option on or off and when the effect is not immediate.
 
-### Label
+-   E.g. if a final confirmation is required to apply the effect.
 
--   Try to limit labels to one short line.
--   Add instructions that indicate how many items can be selected, as some users might not know the difference between checkboxes and radio buttons.
--   Consider listing options vertically, as horizontal listings can make it difficult to tell which label is related to which checkbox.
--   [Partially selected states](#/components/Checkbox) should only be used when sub-options can be selected as children of a checkbox. A parent checkbox being in the partially selected state indicates that only some of its sub-options are selected, not all of them. Toggling the parent checkbox toggles all its sub-options.
+Try to limit the number of available options under 7. If impossible, consider breaking them down into smaller logical sets or use the [Multi-Select](#/form/MultiSelect)
 
-### Behavior
+Options should be ordered by most relevant unless a more suited ordering rational applies (such as for sizes or security levels). If no rational particularly stands-out, use alphanumerical order.
 
--   As much as possible, preselect the recommended option. If there is none, preselect the option most likely to be selected by users.
+## Labeling
+
+Keep titles and labels short, preferably **under three words**.
+
+### Title
+
+Always use a title if you have multiple options.
+
+Use descriptive labels without action verbs.
+
+-   E.g. Use "Grocery items" instead of "Select all items for your grocery".
+
+For stand-alone checkbox, the title is optional. Use a title only if you need to dissociate the stand-alone checkbox from other adjacent options.
+
+### Labels
+
+Labels are used to name each option and should be self-explanatory as much as possible.
+
+Use a consistent writing style for all options in the list.
+
+For stand-alone checkbox, use a positive and active wording that clearly indicates what will happen if the checkbox is turned on.
+
+-   E.g. Use "Send me a text notification." instead of "Email notification".
+
+## Help text and instructions
+
+Instructions provide more context on the outcome of the choice at-hand. Use it sparingly, only when it is critical to the user's understanding.
+
+## Feedback and validation
+
+If you have a maximum number of options to select, add a description text to indicate it.
+
+Indicate which options from the list are the default or recommended one in their labels
+
+Partially selected states should only be used when sub-options can be selected as children of a checkbox. A parent checkbox being in the partially selected state indicates that only some of its sub-options are selected.
```

Comment:
@kpoulincoveo I'm not sure what a partially selected state is? Can you show me what it looks like? I just want to be sure I understand this correctly before I make any changes 😅

Reply:
partial state is this one
![image](https://user-images.githubusercontent.com/44175746/152013084-6baf4d4c-a965-42c3-b566-e90cac3bb19f.png)

It's rarely used in the platform right now if I am not mistaken

Reply:
```suggestion
The partially selected state is only allowed with checkboxes that have children options. A partially selected parent checkbox indicates that only some of its children options are selected.
```